### PR TITLE
Small fix in the "pay with crypto section"

### PR DESCRIPTION
**Pull request by TwoSixtyThreeFiftyFour**
Removed outdated info
Added ref to updated info

### CODE REVIEW

File: guide.md [hunk 1]
```diff
@@ -11836,9 +11836,9 @@ But what if the service you want does not accept Monero but does accept a more m
...
+-   **Stay away from Crypto Mixer, Tumblers and Coinjoiners.** You might think this is a good idea but not only are they useless with cryptocurrencies such as BTC/ETH/LTC, but they are also dangerous as you might end up trading your currency for dirty currency from illicit activities. Use Monero to anonymize your crypto. Use a normal KYC-enabled Exchange to buy/sell your Monero (such as Kraken) or (at your own risk), use a service like LocalMonero.
 
--   **Stay away from what are in my opinion risky private/anonymizing wallets such as <https://we.incognito.org>.** Use a safer method outlined below.
+-   **Have a look below a couple pages below at "Warning about special tumbling, mixing, coinjoining privacy wallets and services" before proceeding.**
```
Comment (pterocles) [inline guide.md:11841]: ?

Comment (NobodySpecial256) [reply guide.md:11841]: The reference doesn't render properly. See https://github.com/Anon-Planet/thgtoa/blob/pr/162/guide.md#appendix-z-online-anonymous-payments-using-cryptocurrencies

Comment (pterocles) [reply guide.md:11841]: Table of Contents:
```suggestion
    -   [Warning about special tumbling, mixing, coinjoining privacy wallets and services] 
```
Line 11841:
```suggestion
-   **See [Warning about special tumbling, mixing, coinjoining privacy wallets and services].**
```

Line 13727:
```suggestion
  [Warning about special tumbling, mixing, coinjoining privacy wallets and services]: #warning-about-special-tumbling-mixing-coinjoining-privacy-wallets-and-services-wikiless-archiveorg 
```

**Review by pterocles — CHANGES_REQUESTED** [review]
Fix pointers and TOC like this.

File: guide.md [hunk 2]
```diff
@@ -11912,7 +11912,7 @@ For example, here are two possibilities:
 
 These steps should upgrade from "reasonably anonymous" to "extra-paranoid anonymous". Even if Monero is broken in the future. Zcash will have to be broken as well. Quite unlikely.
 
-## Warning about special tumbling, mixing, coinjoining privacy wallets and services: <sup>[Wikiless](https://wikiless.org/wiki/Cryptocurrency_tumbler) [Archive.org](https://web.archive.org/web/https://wikiless.org/wiki/Cryptocurrency_tumbler)</sup>
+## [Warning about special tumbling, mixing, coinjoining privacy wallets and services]: <sup>[Wikiless](https://wikiless.org/wiki/Cryptocurrency_tumbler) [Archive.org](https://web.archive.org/web/https://wikiless.org/wiki/Cryptocurrency_tumbler)</sup>
```
Comment (pterocles) [inline guide.md:11915]:
```suggestion
## Warning about special tumbling, mixing, coinjoining privacy wallets and services: <sup>[Wikiless](https://wikiless.org/wiki/Cryptocurrency_tumbler) [Archive.org](https://web.archive.org/web/https://wikiless.org/wiki/Cryptocurrency_tumbler)</sup>

```
Header doesn't need the brackets.

**Review by pterocles — CHANGES_REQUESTED** [review]

**Review by pterocles — APPROVED** [review]
OK looks great!

**Pull request merged**